README.md

```diff
@@ -80,7 +80,13 @@ lerna run start --scope=[module-name]
   },
 ```
 
-- Add entry to `modules.json`
+<<<<<<< HEAD
+
+- # Add entry to `modules.json`
+
+* Add entry to `modules.json`
+
+> > > > > > > 996d0d8fbcc71ecb7ac140c68988fb6878f24ed5
```

Comment:
This is not resolved yet? This was a conflict please fix this.

Reply:
it is resolved i just. took a pull from upstream main and there was no conflit